cli/types/cypress.d.ts

```diff
@@ -2001,7 +2006,7 @@ declare namespace Cypress {
     })
     ```
      */
-    window(options?: Partial<Loggable & Timeoutable>): Chainable<Window>
+    window(options?: Partial<Loggable & Timeoutable>): Chainable<Window & typeof globalThis & ApplicationWindow>
```

Comment:
why add `globalThis`?

Reply:
See https://github.com/cypress-io/cypress/pull/6624

Reply:
I copied and opened the issue at #7856.